src/pages/Homepage.tsx

```diff
@@ -26,27 +26,26 @@ const Homepage = (): ReactElement => {
       name: e.name,
       location: e.location,
       description: e.description,
-      color: getColor(new Date(e.startTime), new Date(e.endTime)),
+      status: getStatus(new Date(e.startTime), new Date(e.endTime)),
     };
   });
 
-  const liveEvents = events
-    .filter((e) => e.color === "bg-sga-red")
-    .map((e) => (
-      <>
-        <EventCard {...e} />
-        <hr className="border-black home-mx" />
-      </>
-    ));
+  const liveEvents: ReactElement[] = [];
+  const upcomingEvents: ReactElement[] = [];
 
-  const upcomingEvents = events
-    .filter((e) => e.color !== "bg-sga-red")
-    .map((e) => (
-      <>
-        <EventCard {...e} />
-        <hr className="border-black home-mx" />
-      </>
-    ));
+  // Filters events into their corresponding status.
+  events.reduce(
```

Comment:
also changing from two filter()s to a reduce() is SO CLEAN your brain is gigantic

Reply:
TYY hate seeing so many filters!!!